bower.json

```diff
@@ -24,13 +24,16 @@
     "tests"
   ],
   "dependencies": {
-    "ember": "^2.7.0"
+    "ember": "components/ember#canary"
```

Comment:
No, it's not. I just wanted to share a branch someone could clone and run to see the error with ember-canary. Something very weird is happening at build/transpile level that `emberBabel.classCallCheck` is undefined. 

I plan to remove this second commit.